File: 2709.java
import java.util.Scanner;
import java.lang.Math;

class Main{
    public static void main(String ...args){
        int moedas [];
        int n,qtdM, soma = 0, ultimoVal=0;
        double maxdiv;
        boolean a=true, t=false;
        
        
        Scanner scan = new Scanner(System.in);
        qtdM = scan.nextInt();
        
        moedas = new int[qtdM];
        
        for (int i=0;i<qtdM;i++)
        {
            moedas[i] = scan.nextInt();
        }
        n = scan.nextInt();
        soma = moedas[qtdM-1];
        //ultimoVal = moedas[qtdM-1];
        for (int i=qtdM-1;i>=0;i = i-n){
                if((i>=0) && (i!=qtdM-1)){
                    soma = soma + moedas[i];
                //    ultimoVal = moedas[i];
                }
        
        }
        
        if(scan.hasNextInt()){
            t = true;
        }
        maxdiv = Math.sqrt(soma);
        //System.out.println(maxdiv);
        int max = (int) (maxdiv + 1);
        for (int i=2;i<=max;i++){
            if(soma==i){
            }
            else{
                if(soma%i==0){
                    System.out.println("Bad boy! I\'ll hit you.");
                    a=false;
                    break;
                }
            }
        }
        if(soma==1){System.out.println("Bad boy! I\'ll hit you.");}
        else{
            if(a)  System.out.println("You\'re a coastal aircraft, Robbie, a large silver aircraft.");
       }
       if(t) System.out.println("Bad boy! I\'ll hit you.");
    }

}

File: 2709.c
#include <stdio.h>
#include <math.h>
 
int main(void) {
        int n,qtdM, soma = 0, ultimoVal=0;
        double mdiv;
        int a=1;
        
        
        scanf ("%d", &qtdM);
       
        
        int moedas[qtdM];
        int i;
        for (i=0;i<qtdM;i++)
        {
        		
            scanf ("%d", &moedas[i]);
        }
        scanf ("%d", &n);
        soma = moedas[qtdM-1];
        //ultimoVal = moedas[qtdM-1];
        int k;
        for (k=qtdM-1;k>=0;k = k-n){
                if((k>=0) && (k!=qtdM-1)){
                    soma = soma + moedas[k];
                //    ultimoVal = moedas[i];
                }
        
        }
        mdiv = sqrt(soma);
        //System.out.println(maxdiv);
        int maxd = (int) (mdiv + 1);
        int j;
        for (j=2;j<=maxd;j++){
            if(soma==j){
            }
            else{
                if(soma%j==0){
                    printf("Bad boy! I'll hit you.");
                    a=0;
                    break;
                }
            }
        }
        if(soma==1){printf("Bad boy! I'll hit you.");}
        else{
            if(a)  printf("You're a coastal aircraft, Robbie, a large silver aircraft.");
       }
   //getch();
	return 0;
}
